node/errors.js

```diff
@@ -445,34 +444,71 @@ module.exports.classify = function classify(err) {
         case 'tchannel.max-pending-for-service':
             return 'Declined';
 
-        case 'tchannel.timeout':
+        case 'tchannel.socket-local-closed':
+        case 'tchannel.local.reset':
+            return 'Cancelled';
```

Comment:
The only time these happen is when you do a graceful shutdown of a connection.

This generally only happens when you call channel.close() and that's only really done by our tests; not in production.

I agree that cancellation is not ideal. But manual closing has nothing to do with network error. As far as we know the network is healthy. Maybe it's better marked as declined due to graceful close.